pluses-echo-server/src/main/main.iml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module type="JAVA_MODULE" version="4">
+  <component name="NewModuleRootManager" inherit-compiler-output="true">
+    <exclude-output />
+    <content url="file://$MODULE_DIR$">
+      <sourceFolder url="file://$MODULE_DIR$/java" isTestSource="false" />
+    </content>
+    <orderEntry type="inheritedJdk" />
+    <orderEntry type="sourceFolder" forTests="false" />
+  </component>
+</module>
```

Comment:
System file of IDEA... drop it

Reply:
Dropped in https://github.com/AlienCoffee/Pluses-Progress-Java-Server/commit/115158e2b6d8f8e79a368e866b9464c313b4ec06